storage/integration_test.go

```diff
@@ -589,6 +589,9 @@ func TestIntegration_BucketUpdate(t *testing.T) {
 		if !testutil.Equal(attrs.Labels, wantLabels) {
 			t.Fatalf("add labels: got %v, want %v", attrs.Labels, wantLabels)
 		}
+		if !attrs.Created.Before(attrs.Updated) {
+			t.Fatal("attrs.Updated should be newer than attrs.Created")
```

Comment:
Nit: These checks should be `Errorf` rather than `Fatalf` because they don't need to stop the execution of the remainder of the test.

Also, include both timestamps in the error for debugging purposes, something like:
`got attrs.Updated %v before attrs.Created %v, want Attrs.Updated to be after`

Reply:
Have changed up.

Reason behind picking Fatalf was that Bucket Attributes should update given that this was not done hence i thought tests should stop. Also all other checks where fatalf only.